spec/requests/availability_reports_request_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+include Devise::Test::IntegrationHelpers
```

Comment:
inclusion should not happen here, they are to be done in the `spec/spec_helper.rb`

Reply:
I tried that. It didn't help...will try at line 99.